develop-env/develop_py36/salobj_4/Dockerfile

```diff
@@ -60,7 +60,7 @@ RUN source /opt/lsst/software/stack/loadLSST.bash && setup lsst_distrib && \
     source /home/saluser/repos/ts_sal/setup.env && \
     setup ts_sal -t current && \
     setup ts_idl -t current && \
-    make_idl_files.py `ls $TS_XML_DIR/sal_interfaces | grep -v xml| tr '\n' ' '`
+    make_idl_files.py `ls $TS_XML_DIR/sal_interfaces | grep -v xml | grep -v MTM1M3| tr '\n' ' '`
```

Comment:
Are we still developing under Python 3.6?

Reply:
No, this came up long ago when we where still switching from 3.6 to 3.7 and I had no idea how to parameterize the build. If you don't mind, I'd like more time to clean this up in a future ticket.

Reply:
As long as there is a ticket filed for the work, I'm fine with this.